src/main/java/com/microsoft/jenkins/appservice/commands/DockerDeployCommand.java

```diff
@@ -85,11 +91,19 @@ public void execute(final IDockerDeployCommandData context) {
                 azure.webApps().inner().updateConfigurationSlot(
                         webApp.resourceGroupName(), webApp.name(), slot.name(), siteConfigResourceInner);
             }
-            context.setDeploymentState(DeploymentState.Success);
+            context.setCommandState(CommandState.Success);
             context.logStatus("Azure app service updated successfully.");
+            AzureAppServicePlugin.sendEvent(Constants.AI_WEB_APP, Constants.AI_DOCKER_DEPLOY,
+                    "WebApp", AppInsightsUtils.hash(context.getWebApp().name()),
```

Comment:
Also include subscription ID and resource group name

Reply:
Added resource group name. Subscription ID is included by default inside the AppInsight SDK.

Reply:
Sorry, misunderstood it. Will add subscription ID and job ID soon.